en/firefox/browsers/compare/brave.ftl

```diff
@@ -55,6 +55,9 @@ compare-brave-the-ability-to-sync-your-passwords = The ability to sync your pass
 #   $accounts (string) - link to /firefox/accounts/ with additional attributes for analytics
 #   $monitor (string) - link to monitor.firefox.com with additional attributes for analytics
 #   $breaches (string) - link to monitor.firefox.com/breaches with additional attributes for analytics
+compare-brave-the-firefox-browser-also-gives-v2 = The { -brand-name-firefox } browser also gives users the ability to sign up for a free <a { $accounts }>{ -brand-name-mozilla-account }</a>. Having a { -brand-name-mozilla-account } is the key to unlocking syncing across devices, plus you get the added benefit of products like <a { $monitor }>{ -brand-name-firefox-monitor }</a> which monitors your email addresses and alerts you if any of your information has been involved in any known <a { $breaches }>data breaches</a>.
```

Comment:
Should this be `{ -brand-name-mozilla-monitor }`?

Reply:
So I was told that the Mozilla Monitor update coming up won't be launched until December 1st, so I'm going to address those string changes in another PR sometime later closer to the date, maybe in late October to get it ready for translations.

Reply:
We want to progressively get rid of them, because they make translation unnecessarily complicated (and Mozilla.org is the only project who set them up at some point).

Not sure where the conversation landed, since I'm normally not directly involved in this project
https://github.com/mozilla-l10n/www-l10n/pull/345

Reply:
Sure, happy to take on the changes, but the current changes I'm addressing are to do with Mozilla accounts. Since the official rebranding of the name Mozilla Monitor happens December 1st, I understood that I should hold off before having those changes happen this early throughout the mozilla.org website

Reply:
Sorry, my last comment somehow ended up in the wrong thread. I wasn't referring to Monitor, but to non-Mozilla brands. I was answering [this comment](https://github.com/mozilla-l10n/www-l10n/pull/350#discussion_r1332982533)

Reply:
Oh thanks for clarifying! That makes sense with new strings going forward. I'll update this now